packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -160,12 +161,20 @@ class FlutterCommandRunner extends CommandRunner {
     return '.';
   }
 
+  Future<dynamic> run(Iterable<String> args) {
+    return super.run(args).then((dynamic result) {
+      if (context[Logger] is VerboseLogger)
```

Comment:
you could put a flush in every logger and just override the one in the verbose logger to do something useful, avoiding the is check here